live-streaming/live-stream-best-practices.md

```diff
@@ -38,49 +31,73 @@ After this, take network connection into consideration - if you or your users st
 
-Make sure to use an outro (static picture, music and etc.) at the end of the stream for 30 seconds. This is a good way for users to indicate that the stream has ended.
+## Limitations
 
-When reaching the end of the playback, the player buffers until the live stream is deleted, which could take from 10 seconds to 5 minutes and could cause the live stream not to end properly even if you have stopped the broadcast.
+This section lists the limitations of api.video's live streaming platform.
 
-## Connectivity loss reconnection
+### Codecs
 
-Reconnection is handled by api.video, however, an edge case might occur (very slim chance), which will result in the inability to reconnect to the stream with good quality. In this case, we recommend creating a manual stream reconnection, where the streamer will create a new streaming key while the consumers will have to refresh their player instance.
+* The **video codec must be H.264**.
+* The **audio codec must be AAC or MP3**.
 
-## Recommended setting for ingestion
+### Live stream recording
 
-Video codec: H.264  
-Audio codec: AAC/MP3  
-Bitrate encoding: CBR  
-Keyframe Interval: 2 second
+Live streams are recorded automatically. The length of each recording is fixed at 12 hours. When a recording reaches this length, a new recording is started.
 
-| Quality  | Framerate | Video bitrate              | Audio sample rate | Audio Bitrate |
-| -------- | --------- | -------------------------- | ----------------- | ------------- |
-| 240p     | 25-30 fps | 300-700 Kbps               | 44,1 kHz          | 64 Kbps       |
-| 360p     | 25-30 fps | 400-1000 Kbps              | 44,1 kHz          | 128 Kbps      |
-| 480p     | 25-30 fps | 500-2000 Kbps              | 44,1 kHz          | 128 Kbps      |
-| 720p     | (30 fps)  | 25-30 fps 1500-4000 Kbps   | 44,1 kHz          | 128 Kbps      |
-| 720p     | (60 fps)  | 60 fps 2250-6000 Kbps      | 44,1 kHz          | 128 Kbps      |
-| 1080p    | (30 fps)  | 25-30 fps 3000-6000 Kbps   | 44,1 kHz          | 128 Kbps      |
-| 1080p    | (60 fps)  | 60 fps 4500-9000 Kbps      | 44,1 kHz          | 128 Kbps      |
-| 2160(4K) | (30 fps)  | 25-30 fps 13000-34000 Kbps | 44,1 kHz          | 192 Kbps      |
-| 2160(4K) | (60 fps)  | 60 fps 20000-51000 Kbps    | 44,1 kHz          | 192 Kbps      |
+### DVR
 
-## Limitations
+When a live stream is ongoing, viewers can replay earlier content with the DVR feature.
+
+* The available duration for DVR is 1 hour.
+* DVR is only available for live streams that are actively broadcasting.
+* When a live stream is completed, api.video stores the stream for another 4 to 5 minutes. The cached DVR is subsequently deleted after the live stream has ended.
+
+### Re-connecting to a live stream
+
+During a disconnection on ingest side, re-connection must occur within 10 seconds.
```

Comment:
Not true anymore, now it is 1 minute to reconnect

Reply:
Fixed ✅